doc/tutorials/video-series-overview.md

```diff
@@ -0,0 +1,62 @@
+# QGreenland Beginner’s Tutorial Video Series Overview:
+
+Welcome to the QGreenland Beginner’s Tutorial Series. In this series, you will become familiar with QGIS, download and browse the QGreenland Core data package, and be introduced to the processing toolbox for basic geospatial data analysis.
+
+This tutorial requires that QGIS is installed on your computer. Go to [qgis.org](https://www.qgis.org/) and download the free QGIS software. QGIS version 3.16 is the oldest version supported by QGreenland.
+
+You can download the QGreenland Core data package at [https://qgreenland.org/download](https://qgreenland.org/download). Go to [Getting started with QGreenland Core](https://qgreenland.readthedocs.io/en/latest/tutorials/get-started.html) for a tutorial and more information.
+
+## The tutorial series contains 7 sessions. The goals of each session are as follows:
+
+### Session 1: Introduction to QGreenland
+- Understand what geospatial data is
+
+Link to Youtube video: [Session 1](https://www.youtube.com/watch?v=gD0vkP5JUmA&list=PLSRiyMridUCwyu-vqpAFtm8bVERgTvs7q&index=1)
```

Comment:
The following sessions have `Link to video: ...` instead of `Link to Youtube video: ...`. Lets make this consistent and just drop the `Youtube` from this bit of text.